config/default.go

```diff
@@ -11,6 +11,8 @@ MaxSQLConnections = 100
 SQLConnectionTimeout = "2s"
 ReadTimeout = "30s"
 WriteTimeout = "30s"
+CoordinatorNetwork = true
+FindPeersCoordinatorNetworkInterval = "180s"
```

Comment:
Please add those fields to the readme and assign them an env name to allow a user configure them using environmental variables